testing/tests/DevExpress.ui.widgets.editors/autocomplete.tests.js

```diff
@@ -657,6 +657,46 @@ QUnit.module('dxAutocomplete', {
         });
     });
 
+    QUnit.test('popup should be opened after press alt + arrow down', function(assert) {
+        const keyboard = this.keyboard;
+        const keyDownConfig = { key: KEY_DOWN, altKey: true };
+
+        keyboard.keyDown(keyDownConfig.key, keyDownConfig);
+
+        assert.strictEqual(this.instance._popup.option('visible'), true);
```

Comment:
Let's not use private api in tests.
I guess u can just check `this.instance.option('opened)'`